src/coral_models/prepare_raw_data.py

```diff
@@ -178,7 +186,7 @@ def make_recording_metadata(
     recording_metadata["filename"] = recording_metadata.index.astype(str)
 
     # Load speaker information from read aloud data
-    recording_metadata_list = [recording_metadata]
+    recording_metadata_list = []
     read_aloud_paths = raw_path.glob("*_oplæst_*")
     for read_aloud_path in tqdm(list(read_aloud_paths)):
```

Comment:
This entire `for` block (and the following couple of code blocks with post-processing), concerning the construction of metadata for read-aloud samples, should probably be a separate function. This one is getting a bit bloated.

Reply:
Refactoring in another PR.